app/code/Transoft/Blog/Controller/Index/Index.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Why do we need this page? What does it stand for?

Reply:
removed